### PR TITLE
Create yml files for samples releases

### PR DESCRIPTION
**Pull request by kacperoak**

### CODE REVIEW

File: .github/workflows/ci_release_sample_dapp.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: SAMPLES RELEASE
+
+on:
+  push:
+    branches: ["release"]
```
Comment (kacperoak) [inline .github/workflows/ci_release_sample_dapp.yml:5]: This is something we can discuss about when we want to trigger these actions

**Comment by kacperoak** [comment]
After sync changing it into draft and try to move it in gradle plugin: 
https://firebase.google.com/docs/app-distribution/android/distribute-gradle?hl=en&authuser=0&apptype=apk

File: .github/workflows/ci_release_sample.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: Sample releases
+
+on:
+  push:
+#    branches: ["master"]
```
Comment (kacperoak) [inline .github/workflows/ci_release_sample.yml:5]: lets talk about trigger

File: .github/workflows/ci_release_sample_dapp.yml (hunk 1 above)
Comment (jakubuid) [reply .github/workflows/ci_release_sample_dapp.yml]: I think we should have a stage between master and develop. I'd say staging that triggers sending samples to testers via firebase. Is this what you meant?

Comment (kacperoak) [reply .github/workflows/ci_release_sample_dapp.yml]: Yes

File: .github/workflows/ci_release_sample.yml (hunk 2 above)
Comment (Elyniss) [inline .github/workflows/ci_release_sample.yml:5]:
```suggestion
on: workflow_dispatch
```

**Review by TalhaAli00 — APPROVED** [review]

**Review by jakubuid — APPROVED** [review]

**Pull request closed without merging**